frontend/src/components/dashboard/campaigns/Campaigns.tsx

```diff
@@ -27,10 +27,12 @@ import { getUserSettings } from 'services/settings.service'
 import DuplicateCampaignModal from '../create/duplicate-campaign-modal'
 import AnnouncementModal from './announcement-modal'
 import { ANNOUNCEMENT, getAnnouncementVersion } from 'config'
+import useIsMounted from 'components/custom-hooks/use-is-mounted'
 
 const ITEMS_PER_PAGE = 10
 
 const Campaigns = () => {
+  const isMounted = useIsMounted()
```

Comment:
This change along with. those in `SMSRecipients.tsx` and `TelegramRecipients.tsx` are explored in greater detail in https://github.com/opengovsg/postmangovsg/pull/1109